test/Microsoft.DotNet.Tools.Publish.Tests/Microsoft.DotNet.Tools.Publish.Tests.cs

```diff
@@ -213,17 +213,26 @@ public void PublishScriptsRun()
             result.Should().Pass();
         }
 
+        private void GenerateTempGlobalJson(TempDirectory tempDir, string repoRoot)
```

Comment:
I did not see a single use of this method that takes anything different from RepoRoot as the second argument. Why do even have it? Replace it with the property in the body of the method.